compiler/luci-interpreter/src/kernels/Logistic.test.cpp

```diff
@@ -25,33 +25,63 @@ namespace
+           float input_min, float input_max, float output_min, float output_max,
+           DataType element_type)
 {
-  Shape input_shape{1, 2, 4, 1};
-  std::vector<float> input_data{
-      0, -6, 2,  4, //
-      3, -2, 10, 1, //
-  };
-  Tensor input_tensor = makeInputTensor<DataType::FLOAT32>(input_shape, input_data);
-  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+  std::pair<float, int32_t> input_quant_params = quantizationParams<T>(input_min, input_max);
+  std::pair<float, int32_t> output_quant_params = quantizationParams<T>(output_min, output_max);
+  std::vector<T> quantized_input_value =
+      quantize<T>(input_data, input_quant_params.first, input_quant_params.second);
+  Tensor input_tensor{
+      element_type, input_shape, {{input_quant_params.first}, {input_quant_params.second}}, ""};
+  input_tensor.writeData(quantized_input_value.data(), quantized_input_value.size() * sizeof(T));
+  Tensor output_tensor =
+      makeOutputTensor(element_type, output_quant_params.first, output_quant_params.second);
 
   Logistic kernel(&input_tensor, &output_tensor);
   kernel.configure();
   kernel.execute();
 
-  std::vector<float> ref_output_data{
-      0.5,      0.002473, 0.880797, 0.982014, //
-      0.952574, 0.119203, 0.999955, 0.731059, //
-  };
-  EXPECT_THAT(extractTensorData<float>(output_tensor),
-              ElementsAreArray(ArrayFloatNear(ref_output_data)));
-  // TODO make a Shape checking of output_tensor.
+  EXPECT_THAT(dequantize(extractTensorData<T>(output_tensor), output_tensor.scale(),
+                         output_tensor.zero_point()),
+              ElementsAreArray(ArrayFloatNear(output_data, kQuantizedTolerance)));
 }
 
-TEST(LogisticTest, Uint8)
+template <typename T> class LogisticTest : public ::testing::Test
 {
-  // Need to Implement GetDequantizedOutput Function.
+};
+
+using DataTypes = ::testing::Types<float, uint8_t>;
+TYPED_TEST_CASE(LogisticTest, DataTypes);
+
+TYPED_TEST(LogisticTest, TotalTest)
+{
+  Check<TypeParam>(/*input_shape=*/{1, 6, 4, 1}, /*output_shape=*/{1, 6, 4, 1},
+                   /*input_data=*/
+                   {
+                       0, -6, 2,  4, //
+                       3, -2, 10, 1, //
+                       0, -6, 2,  4, //
+                       3, -2, 10, 1, //
+                       0, -6, 2,  4, //
+                       3, -2, 10, 1, //
+                   },
+                   /*output_data=*/
+                   {
+                       0.5,      0.002473, 0.880797, 0.982014, //
+                       0.952574, 0.119203, 0.999955, 0.731059, //
+                       0.5,      0.002473, 0.880797, 0.982014, //
+                       0.952574, 0.119203, 0.999955, 0.731059, //
+                       0.5,      0.002473, 0.880797, 0.982014, //
+                       0.952574, 0.119203, 0.999955, 0.731059, //
+                   },
+                   /*input_min=*/-10.0f, /*input_max=*/10.0f, /*output_min=*/0.0f,
```

Comment:
How did you set these min/max values?

Reply:
I referenced from official Tensorflow example.

Reply:
Oh, Found Something wired. I will make a new commit for this. Sorry for confused.